src/components/application_manager/test/commands/hmi/ui_get_capabilities_response_test.cc

```diff
@@ -186,7 +187,7 @@ TEST_F(UIGetCapabilitiesResponseTest, SetNavigation_SUCCESS) {
       CreateCommand<UIGetCapabilitiesResponse>(command_msg));
 
   EXPECT_CALL(app_mngr_, hmi_capabilities())
-      .WillOnce(ReturnRef(mock_hmi_capabilities_));
+      .WillOnce(Return(NonConstDataAccessor<application_manager::HMICapabilities>(mock_hmi_capabilities_, hmi_lock_)));
```

Comment:
@fronneburg I would create typedef for NonConstDataAccessor<application_manager::HMICapabilities>